# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../strace/glibc-2.31.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- strace-5.2/ptrace.h.vanilla	2020-02-05 13:11:15.681357577 +0100
+++ strace-5.2/ptrace.h	2020-02-05 13:11:28.297357584 +0100
@@ -156,7 +156,7 @@
 # ifndef PTRACE_SECCOMP_GET_METADATA
 #  define PTRACE_SECCOMP_GET_METADATA	0x420d
 # endif
-# ifndef PTRACE_GET_SYSCALL_INFO
+# if 1 // ifndef PTRACE_GET_SYSCALL_INFO
 #  define PTRACE_GET_SYSCALL_INFO	0x420e
 #  define PTRACE_SYSCALL_INFO_NONE	0
 #  define PTRACE_SYSCALL_INFO_ENTRY	1
